tools/messages.py

```diff
@@ -39,6 +39,7 @@ def info(
             by the user.
         :param success: Whether the message is success message or not
         """
+        LOGGER.info(str(details), **MsgBar.KWARGS)
```

Comment:
Could you elaborate what these added logger messages does? Are these shown in the message bar?

Reply:
Before details part got logged only to QGIS message bar and not into other handlers. I added those lines to get details logged to other handlers as well.

Reply:
I see it now. And it is not logger twice to the message bar because it has no details extra field? right?
Should it be checked if details is empty string then not to log.
Should we log the details before or after the actual message? My first thought was that details should be logged after.

Reply:
I can add the check there. I initially put it after, but usually the detailed message is actually more important.